packages/nimble-components/src/breadcrumb-item/styles.ts

```diff
@@ -1,81 +1,59 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '../utilities/style/display';
 import {
-    bodyFontColor,
-    borderHoverColor,
-    borderWidth,
     controlHeight,
     iconSize,
-    linkFont,
+    linkDisabledFontColor,
+    mediumPadding,
     placeholderFontColor
 } from '../theme-provider/design-tokens';
-import { focusVisible } from '../utilities/style/focus';
+import { styles as anchorStyles } from '../anchor/styles';
 
 export const styles = css`
-    ${display('inline-flex')}
+    ${anchorStyles}
 
-    :host {
-        height: ${controlHeight};
-        padding-left: calc(4px - ${borderWidth});
+    @layer base {
+        ${display('inline-flex')}
 
-        ${
-            /* When href removed the .control element is also removed
-             so this becomes the fallback color for the slot */ ''
-        }
-        color: ${bodyFontColor};
-        font: ${linkFont};
-    }
-
-    .listitem {
-        display: flex;
-        align-items: center;
-    }
-
-    .control {
-        color: var(--ni-private-breadcrumb-link-font-color);
-        display: flex;
-        align-items: center;
-        justify-content: center;
-        border: ${borderWidth} solid transparent;
-        padding-right: calc(4px - ${borderWidth});
-        text-decoration: none;
-    }
+        :host {
+            height: ${controlHeight};
+            padding-left: ${mediumPadding};
 
-    .control:hover {
-        text-decoration: underline;
-    }
-
-    .control${focusVisible} {
-        border: ${borderWidth} solid ${borderHoverColor};
-        outline: 2px solid ${borderHoverColor};
-        outline-offset: 1px;
-    }
-
-    .control:active {
-        color: var(--ni-private-breadcrumb-link-active-font-color);
-        text-decoration: none;
-    }
+            ${
+                /* When href removed the .control element is also removed
+                so this becomes the fallback color for the slot */ ''
+            }
+            color: ${linkDisabledFontColor};
+        }
 
-    [part='start'] {
-        display: none;
-    }
+        .listitem {
+            display: flex;
+            align-items: center;
+        }
 
-    .content {
-        pointer-events: none;
-    }
+        .control {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            margin-right: ${mediumPadding};
```

Comment:
We now use `box-shadow` to draw the focus underline (comes from the anchor styles). I needed to change this from `padding-right` to `margin-right` so that the `box-shadow` wouldn't extend past the end of the text.

Reply:
Moot now. The padding is on the separator, which simplifies a lot.